app/src/main/res/layout/fragment_video_preview.xml

```diff
@@ -25,18 +25,54 @@
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_alignParentStart="true"
-                android:layout_alignParentLeft="true"
                 android:layout_alignParentTop="true"
                 android:textColor="@color/text_main"
                 android:textSize="24sp" />
 
+            <TextView
+                android:id="@+id/textSubtitles"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/textTitle"
+                android:ellipsize="marquee"
+                android:singleLine="true"
+                android:textColor="@color/text_second"
+                android:textSize="14sp"
+                android:visibility="gone" />
+
+            <FrameLayout
+                android:id="@+id/videoDescriptionContainer"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/textSubtitles"
+                android:layout_marginTop="8dp"
+                android:foreground="@drawable/gradient_light_to_transparent_from_bottom">
+
+                <TextView
+                    android:id="@+id/textPreviewDescription"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:ellipsize="end"
+                    android:maxLines="3"
```

Comment:
Does this need to be set statically or is it possible to set it programmatically to our constant?